StoreManagementService/src/PBJ.StoreManagementService.DataAccess/Entities/PaginationResponse.cs

```diff
@@ -0,0 +1,16 @@
+﻿using PBJ.StoreManagementService.DataAccess.Entities.Abstract;
+
+namespace PBJ.StoreManagementService.DataAccess.Entities
+{
+    public class PaginationResponse<TEntity>
+    where TEntity : BaseEntity
+    {
+        public IEnumerable<TEntity> Items { get; set; } = new List<TEntity>();
```

Comment:
What the reason for 2 almoust duplicated models? Please leave just one and make it totally generic for both options (without relations fot TEntity and TDto)